Sources/SwiftRefactor/ConvertComputedPropertyToStored.swift

```diff
@@ -0,0 +1,55 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if swift(>=6)
+public import SwiftSyntax
+#else
+import SwiftSyntax
+#endif
+
+public struct ConvertComputedPropertyToStored: SyntaxRefactoringProvider {
+  
+  public static func refactor(syntax: VariableDeclSyntax, in context: ()) -> VariableDeclSyntax? {
+    guard syntax.bindings.count == 1, let binding = syntax.bindings.first,
+          let accessorBlock = binding.accessorBlock, case let .getter(body) = accessorBlock.accessors, !body.isEmpty else {
+      return nil
+    }
+    
+    var initializer: InitializerClauseSyntax?
+    
+    if body.count > 1, let closure = body.with(\.trailingTrivia, .newline).as(ClosureExprSyntax.self) {
+      initializer = InitializerClauseSyntax(
+        value: FunctionCallExprSyntax(
+          leadingTrivia: .space,
+          calledExpression: closure,
+          leftParen: .leftParenToken(),
+          arguments: {[]}(),
+          rightParen: .rightParenToken()
+        )
+      )
+    } else if let item = body.first?.item.as(ExprSyntax.self) {
+      initializer = InitializerClauseSyntax(
+        value: item.with(\.leadingTrivia, .space).with(\.trailingTrivia, Trivia())
+      )
+    }
+    
+    guard let initializer else { return nil }
+    
+    let newBinding = binding
+      .with(\.initializer, initializer)
+      .with(\.accessorBlock, nil)
+    
+    return syntax
+      .with(\.bindingSpecifier, .keyword(.let, trailingTrivia: .space))
```

Comment:
I think we should retain the trivia of the old binding specifier. I think with the current implementation, we would drop any doc comments on the variable declaration because they are attached to the binding specifiers (since it’s the first token in the variable declaration).